packages/server/src/integrations/airtable.ts

```diff
@@ -20,7 +20,9 @@ const SCHEMA: Integration = {
     "Airtable is a spreadsheet-database hybrid, with the features of a database but applied to a spreadsheet.",
   friendlyName: "Airtable",
   type: "Spreadsheet",
-  features: [DatasourceFeature.CONNECTION_CHECKING],
+  features: {
+    [DatasourceFeature.CONNECTION_CHECKING]: true,
```

Comment:
What is the reason for the conversion from an array of features supported, to a map of booleans? Is there a benefit to doing it this way?

Only reason I ask is that this style of feature management (via an array) has been implemented into automations as part of Peter's work towards sync automations - just want to check if we should be refactoring it as well if there is a benefit.

Reply:
The main reason I changed that is for the way we consume it. This features will be mainly a lookup, and `features[Feature.X]` is more efficient and clearer than `features.find(x=>Feature.X)`.
If that's a problem we can keep it as an array and reduce it when required